### PR TITLE
Remove use of 'actions-cache-org-ui' feature version

### PR DESCRIPTION
**Pull request by peterbe**
### Why:

`data/features/actions-cache-org-ui.yml` looks like this:

```
# Reference: #8112
# Documentation for the Actions cache list UI for organizations
# Related feature flags are actions-cache-admin-ui and actions-cache-ui
# All features released at the same time, but have different versions
versions:
  fpt: '*'
  ghec: '*'
  ghes: '>= 3.8'

```

So it's always true in all current possible versions. 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Removed an `ifversion` Liquid block that will always resolve to true. 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
Thank you @peterbe for this update! I'll update the branch and get this merged once tests are passing 🍏

**Comment by peterbe** [comment]
@nguyenalex836 This PR is similar to many alike that I've done in docs-internal. 
For example
- https://github.com/github/docs-internal/pull/51142
- https://github.com/github/docs-internal/pull/51129
- https://github.com/github/docs-internal/pull/51122
- https://github.com/github/docs-internal/pull/51121

**Review by ratchanonsuttawas — COMMENTED** [review]
>Review these changes

**Comment by ratchanonsuttawas** [comment]
> 
> 
> ### Why:
> 
> `data/features/actions-cache-org-ui.yml` looks like this:
> 
> ```
> # Reference: #8112
> # Documentation for the Actions cache list UI for organizations
> # Related feature flags are actions-cache-admin-ui and actions-cache-ui
> # All features released at the same time, but have different versions
> versions:
>   fpt: '*'
>   ghec: '*'
>   ghes: '>= 3.8'
> 
> ```
> 
> So it's always true in all current possible versions. 
> 
> 
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> 
> 
> Removed an `ifversion` Liquid block that will always resolve to true. 
> 
> ### Check off the following:
> 
> - [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
> 
>   - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> - [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).
>

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
Comment (ratchanonsuttawas) [inline content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: Approve these changes

**Review by ratchanonsuttawas — COMMENTED** [review]
Summit feedback approve merging these changes

**Review by ratchanonsuttawas — COMMENTED** [review]
Review these changes

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > Approve these changes

Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: Leave a reply

Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > Leave a reply

Comment (ratchanonsuttawas) [inline content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: Approve

**Review by ratchanonsuttawas — COMMENTED** [review]
1 pending review comments

**Comment by ratchanonsuttawas** [comment]
> Review these changes

**Comment by ratchanonsuttawas** [comment]
> Review these changes

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > >Review these changes

**Comment by ratchanonsuttawas** [comment]
> Review these changes

**Comment by ratchanonsuttawas** [comment]
> Review these changes

**Comment by ratchanonsuttawas** [comment]
> Review these changes

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > >Review these changes

**Comment by ratchanonsuttawas** [comment]
> Summit feedback approve merging these changes

**Comment by ratchanonsuttawas** [comment]
>review these changes 100022165870ca645bf53d2a58a895c498fd87f1

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: Reply

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > Reply

Comment (ratchanonsuttawas) [reply content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: > >Approve these changes +0 -4

**Comment by ratchanonsuttawas** [comment]
> 
> 
> ### Why:
> 
> `data/features/actions-cache-org-ui.yml` looks like this:
> 
> ```
> # Reference: #8112
> # Documentation for the Actions cache list UI for organizations
> # Related feature flags are actions-cache-admin-ui and actions-cache-ui
> # All features released at the same time, but have different versions
> versions:
>   fpt: '*'
>   ghec: '*'
>   ghes: '>= 3.8'
> 
> ```
> 
> So it's always true in all current possible versions. 
> 
> 
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> 
> 
> Removed an `ifversion` Liquid block that will always resolve to true. 
> 
> ### Check off the following:
> 
> - [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
> 
>   - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> - [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).
>

File: content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md
Comment (ratchanonsuttawas) [inline content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: Review these changes

Comment (ratchanonsuttawas) [inline content/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization.md:1]: >Review these changes 100022165870ca645bf53d2a58a895c498fd87f1

**Pull request merged**